INSTALL.rst

```diff
@@ -1,123 +1,104 @@
 Installing scikit-image
 =======================
 
-We are assuming that you have default Python environment already configured on
-your computer and you intend to install ``scikit-image`` inside of it. If you
-want to create and work with Python virtual environments, please follow the
-instructions on `venv`_ and `virtual environments`_.
-
-There are two ways you can install ``scikit-image`` on your preferred Python
-environment.
-
-1. Standard Installation
-2. Development Installation
-
-1. Standard Installation:
--------------------------
-
-``scikit-image`` comes pre-installed with several Python distributions,
-including `Anaconda <https://www.anaconda.com/download/>`_,
-`Enthought Canopy <https://www.enthought.com/product/canopy/>`_,
-`Python(x,y) <https://python-xy.github.io/>`_ and
-`WinPython <https://winpython.github.io/>`_.
-
-On all major operating systems, install it via shell/command prompt::
-
+We highlight four different ways to install and use ``scikit-image`` for
+Python3:
+
+1. Pre-installed scientific python distributions (recommended)
+2. Isolated environment based installation (recommended)
+3. Source based installation for development purposes
+4. Using the operating system's package manager (not recommended)
+
```

Comment:
I think it would be better in this case if we maintain the _recommended_ ways to install it, and maintain 3. for devs, removing 4. Maybe we could put a note saying that 4. exists. What do you think?

Reply:
I am +1 with @alexdesiqueira suggestion: simply recall that the option 4 exists but that it is not recommended because the packaged versions are usually not up to date.

Reply:
Lets discuss scoping in https://github.com/scikit-image/scikit-image/issues/4672

It seems we have different views on who should actually read this.